lib/test_helper/event_handler_fake.ex

```diff
@@ -15,6 +15,18 @@ defmodule ElixirTools.TestHelper.EventHandlerFake do
     :ok
   end
 
+  @impl true
+  def publish_event_call(event, _) do
+    send(self(), {:publish_event_call, event})
+    :ok
+  end
+
+  @impl true
+  def publish_event_call(event, schema, _) do
+    send(self(), {:publish_event_call, [event, schema]})
```

Comment:
I think if we add `validate_json_schema(map, event_schema)` to get the parsed schema would be awesome. If not we could also do it previously. But I like better to add it here wdyt?